apps/meteor/app/apps/server/converters/messages.js

```diff
@@ -48,6 +48,7 @@ export class AppMessagesConverter {
 			attachments: getAttachments,
 			sender: 'u',
 			threadMsgCount: 'tcount',
+			...extraFields,
```

Comment:
I think we don't need to be configurable here, you can add the `type: 't'` property directly, not all messages have it anyway.

Also be sure to include the same property to the map in the `convertMessage` method